server/service/integration_enterprise_test.go

```diff
@@ -1171,3 +1179,51 @@ func (s *integrationEnterpriseTestSuite) TestCustomTransparencyURL() {
 	require.NoError(t, deviceResp.Err)
 	require.Equal(t, fleet.DefaultTransparencyURL, rawResp.Header.Get("Location"))
 }
+
+func (s *integrationEnterpriseTestSuite) TestSSOJITProvisioning() {
```

Comment:
Should we have `SAML_IDP_TEST=1` check? (Similar to what we have with `MYSQL_TEST`, `REDIS_TEST`.)

Reply:
I was doubtful, but sounds like a good idea. Added in [11c3ff5](https://github.com/fleetdm/fleet/pull/7182/commits/11c3ff53efd740bc082fe1aacadea9663efd374a)